krupal.txt

```diff
@@ -0,0 +1 @@
+hi from krupal
```

Comment:
write something more about you

Reply:
i have made changes accpording to your request